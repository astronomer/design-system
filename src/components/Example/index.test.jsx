import React from 'react';
import renderer from 'react-test-renderer';

import Example from '.';

test('Example exists', () => {
  const component = renderer.create(
    <Example />,
  );

  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
