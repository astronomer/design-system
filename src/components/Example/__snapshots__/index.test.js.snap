// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Example exists 1`] = `
<div
  className="example"
>
  This is an example by SB
</div>
`;
